store/mockstore/mocktikv/cluster.go

```diff
@@ -464,8 +464,8 @@ func (r *Region) removePeer(peerID uint64) {
 	r.incConfVer()
 }
 
-func (r *Region) changeLeader(leaderStoreID uint64) {
-	r.leader = leaderStoreID
+func (r *Region) changeLeader(leaderID uint64) {
```

Comment:
Is this the ID of a peer or store?

Reply:
peer, it was mistakenly name `leaderStoreID`.